sections/00_title_authors.md

```diff
@@ -2,58 +2,12 @@
-Avanti Shrikumar<sup>14</sup>,
-Jinbo Xu<sup>15</sup>,
-Evan M. Cofer<sup>16</sup>,
-David J. Harris<sup>17</sup>,
-Dave DeCaprio<sup>18</sup>,
-Yanjun Qi<sup>12</sup>,
-Anshul Kundaje<sup>19</sup>,
-Yifan Peng<sup>20</sup>,
-Laura K. Wiley<sup>21</sup>,
-Marwin H.S. Segler<sup>22</sup>,
-Anthony Gitter<sup>23,†</sup>,
-Casey S. Greene<sup>2,†</sup>
+{% for author in authors.authors -%}
+  {{author.name}}<sup>{{author.affiliation}}{{author.symbol}}</sup>{{ ',' if not loop.last }}
+{% endfor %}
 
 <sup>*</sup> Author order was determined with a randomized algorithm
 
-<sup>†</sup> To whom correspondence should be addressed: gitter@biostat.wisc.edu (A.G.) and csgreene@upenn.edu (C.S.G.)
+<sup>†</sup> To whom correspondence should be addressed: {{authors.corresponding}}
 
-1. Molecular Biosciences and Bioengineering Graduate Program, University of Hawaii at Manoa, Honolulu, HI
-2. Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA
-3. Genomics and Computational Biology Graduate Group, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA
-4. Department of Computational Medicine and Bioinformatics, University of Michigan Medical School, Ann Arbor, MI
-5. Harvard Medical School, Boston, MA
-6. Computational Biology and Stats, Target Sciences, GlaxoSmithKline, Stevenage, United Kingdom
-7. Data Science Institute, Imperial College London, London, United Kingdom
-8. Electrical Engineering and Computer Science, Vanderbilt University, Nashville, TN
-9. Ecological and Evolutionary Signal-processing and Informatics Laboratory, Department of Electrical and Computer Engineering, Drexel University, Philadelphia, PA
-10. Computational Biology Department, School of Computer Science, Carnegie Mellon University, Pittsburgh, PA
-11. Biophysics Program, Stanford University, Stanford, CA
-12. Department of Computer Science, University of Virginia, Charlottesville, VA
-13. Imaging Platform, Broad Institute of Harvard and MIT, Cambridge, MA
-14. Department of Computer Science, Stanford University, Stanford, CA
-15. Toyota Technological Institute at Chicago, Chicago, IL
-16. Department of Computer Science, Trinity University, San Antonio, TX
-17. Department of Wildlife Ecology and Conservation, University of Florida, Gainesville, FL
-18. ClosedLoop.ai, Austin, TX
-19. Department of Genetics and Department of Computer Science, Stanford University, Stanford, CA
-20. National Center for Biotechnology Information and National Library of Medicine, National Institutes of Health, Bethesda, MD
-21. Division of Biomedical Informatics and Personalized Medicine, University of Colorado School of Medicine, Aurora, CO
-22. Institute of Organic Chemistry, Westfälische Wilhelms-Universität Münster, Münster, Germany
-23. Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison and Morgridge Institute for Research, Madison, WI
+{{authors.affiliations}}
```

Comment:
The formatting of the affiliation index and the affiliation string (department, university, city)?  That should be pretty easy.